test/plug/builder_test.exs

```diff
@@ -99,13 +99,17 @@ defmodule Plug.BuilderTest do
     assert conn.assigns[:entered_stack] == true
   end
 
-  test "halt/1 halts the plug stack" do
-    conn = conn(:get, "/") |> Halter.call([])
-    assert conn.halted
-    assert conn.assigns[:first]
-    assert conn.assigns[:second]
-    assert conn.assigns[:authorize_reached]
-    refute conn.assigns[:end_of_chain_reached]
+  test "halt/2 halts the plug stack and Logger.infos it" do
+    log = capture_log fn ->
```

Comment:
Unfortunately capture_log messes with global state so we wouldn't be able to leave this test case as async. My recommendation is to move this test to `plug/logger_test`... thoughts?